src/firebase/firebase.js

```diff
@@ -0,0 +1,19 @@
+import * as firebase from "firebase";
+
+const config = {
+  apiKey: "AIzaSyByKLGpoi0Xbu18wNYhPNCoeC6BaqSb5EQ",
```

Comment:
Oh no! Please, move it to the `.env` file. Secure data mustn't be added to the github repository.